lib/ansible/modules/storage/netapp/na_ontap_gather_facts.py

```diff
@@ -207,7 +207,7 @@ def __init__(self, module):
                 'kwargs': {
                     'call': 'lun-get-iter',
                     'attribute': 'lun-info',
-                    'field': 'path',
+                    'field': ('path', 'vserver'),
```

Comment:
I agree it would be better to use vserver as the first element of the key.

Reply:
I've swapped vserver and path order for lun_info field.